spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/wavefront/WavefrontProperties.java

```diff
@@ -36,19 +36,18 @@
 public class WavefrontProperties {
```

Comment:
I'm thinking if we should delete this or keep it.
Also, if we want to keep it should we leave it in this package? Similarly, should we leave the `WavefrontSenderConfiguration` in this package?